src/swarm/neo/client/mixins/AllNodesRequestCore.d

```diff
@@ -140,8 +141,14 @@ public struct AllNodesRequest ( Request, Connector, Disconnected, Initialiser,
 
                 this.handler();
             }
-            // Only retry in the case of a connection error. Other errors
-            // indicate internal problems and should not be retried.
+            // Only retry in the case of a connection I/O error or error event
+            // in epoll. Other errors indicate internal problems and should not
+            // be retried.
+            catch ( IOError e )
```

Comment:
Hmm, the [description](https://github.com/sociomantic-tsunami/ocean/blob/ba54399a79812c2be4c135a0597b88a93113f568/src/ocean/io/select/protocol/generic/ErrnoIOException.d#L34-L35) of `IOWarning` doesn't sound like that, but I'm not sure where it's actually thrown, in practice.